context.go

```diff
@@ -28,3 +29,15 @@ func SpanFromContext(ctx context.Context) (ot.Span, bool) {
 
 	return sp, true
 }
+
+func AddToContext(ctx context.Context, key ContextKey, value string) context.Context {
+	return context.WithValue(ctx, key, value)
+}
+
+func GetValueFromContext(ctx context.Context, key ContextKey) string {
+	val, ok := ctx.Value(key).(string)
+	if !ok {
```

Comment:
This code is redundant! The value of `val` will be an empty string even if `ok` is false. So, we can simply return `val` directly.

Reply:
Good catch 👍. Done!